docs/user/orion.rst

```diff
@@ -10,77 +10,82 @@ Orion
-- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission
-- The Artemis home page if no Artemis project is opened
+- The exercise details page if an exercise is opened as a student.
+- The exercise edit in editor page if an exercise is opened as an instructor.
+- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission.
+- The Artemis home page if no Artemis project is opened.
 
 Settings
 ^^^^^^^^
 
 Orion's settings are at *Settings -> Tools -> Orion*. The settings include:
 
-- The Artemis base url. Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a /, otherwise it will not work!
-- The Artemis exercise paths. Orion will suggest to store newly downloaded exercises at *default path/course/exercise name*, with the default path dependant of the setting.
-- The default commit message
-- An option to change the user agent if needed as well as a button to restart the integrated browser. If Orion gets stuck restarting the browser might solve the issue without restarting IntelliJ.
+- Artemis base url: Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a ``/``, otherwise it does not work!
+- Artemis exercise paths: Orion suggests to store newly downloaded exercises at ``default-path/course/exercise-name``, with the default path dependent of the setting.
+- Default commit message: The default message for each commit.
+- Change user agent: The user agent is sent to Artemis to identify Orion. I needs to contain "Orion/*orion-version*". Usually, no changes are required.
+- Restart the integrated browser: If Orion gets stuck, restarting the browser might solve the issue without restarting IntelliJ.
 
 Problems
 ^^^^^^^^
 
-Please report any issues on `GitHub <https://github.com/ls1intum/Orion>`_.
+Please report any issues on `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_.
 
-Orion for Students
-------------------
+Participation in Orion
+----------------------
 
 Students can participate in programming exercises via Orion by performing the following steps:
 
  1. Navigate to the exercise using the integrated browser.
- 2. After starting the exercise, a button to |open-in-intellij-button| appears.
+ 2. After starting the exercise, click |open-in-intellij-button|.
  3. After cloning, the exercise can be solved in IntelliJ.
- 4. Orion provides a button to |submit-button|. This will automatically commit and push the local changes and display the result of the automatic tests in IntelliJ.
+ 4. To submit the changes, click |submit-button|. This automatically commits, pushes the local changes, and displays the result of the automatic tests in IntelliJ.
```

Comment:
```suggestion
 4. To submit the changes, click |submit-button|. This automatically commits and pushes the local changes and displays the result of the automatic tests in IntelliJ.
```
Sorry, I now realised that the and is required since commit also relates to the local changes.